tests/components/recorder/test_entity_registry.py

```diff
@@ -38,8 +37,13 @@ def _count_entity_id_in_states_meta(
     )
 
 
+@pytest.fixture(autouse=True)
+def enable_recorder(recorder_mock: Recorder) -> recorder.Recorder:
```

Comment:
It's `async_setup_recorder_instance` that we want to setup before `hass`. Autousing `recorder_mock` won't help since it depends on `hass`.

Reply:
It does help until pytest makes a breaking change in the order in which it sets up fixtures.